2023/incoming/dispatch-fu.pod

```diff
@@ -0,0 +1,196 @@
+Author: oodler@cpan.org
+Title: Santa's New Dispatcher
+Topic: Dispatch::Fu
+
+St. Nick's had computer problems nearly non-stop this year, and at
+a critical time he's been unable to access any letters "To Santa"
+that have been emailed to him, which is quite a few considering
+this Internet fad doesn't seem to be slowing down any time soon.
+
+To that end, at this critical hour he's decided to write a Perl
+script that will automatically assign a toy to a child based on
+the starting letter of their first name and the random flip of a
+coin.
+
+At first pass, his code seems to do what he wants; but it's a nasty
+set of C<if-elsif-else> statements. Mrs. Clause would most certainly
+not find it very nice!
+
+=encoding utf8
+
+=begin perl
+
+use strict;
+use warnings;
+
+my (@teddybear, @rockinghorse, @jackinthebox, @gijoe, @barbie, @sallytalksalot, @teaset, @dollhouse);
+
+sub get_child();
+
+foreach my $CHILD ( get_child() ) {
+  if ($CHILD =~ m/^[a-fA-F]/) {
+    if (int rand 2 == 0) {
+      push @teddybear, $CHILD;
+    }
+    else {
+      push @dollhouse, $CHILD;
+    }
+  }
+  elsif ($CHILD =~ m/^[g-lG-L]/) {
+    if (int rand 2 == 0) {
+      push @rockinghorse, $CHILD;
+    }
+    else {
+      push @teaset, $CHILD;
+    }
+  }
+  elsif ($CHILD =~ m/^[m-rM-R]/) {
+    if (int rand 2 == 0) {
+      push @jackinthebox, $CHILD;
+    }
+    else {
+      push @sallytalksalot, $CHILD;
+    }
+  }
+  elsif ($CHILD =~ m/^[s-zS-Z]/) {
+    if (int rand 2 == 0) {
+      push @gijoe, $CHILD;
+    }
+    else {
+      push @barbie, $CHILD;
+    }
+  }
+}
```

Comment:
Trying to run this code I get `Undefined subroutine &main::get_child`

Reply:
looking ...

Reply:
done in latest PR